config/locales/forms/en.yml

```diff
@@ -9,17 +9,19 @@ en:
         - We’ll give you 10 codes that you can download, print, copy or write
           down. You’ll enter one code every time you sign in.
       are_you_sure_title: Are you sure you want to use backup codes?
+      caution_codes: Each code can only be used once. We’ll give you new codes after
+        you use all ten.
       caution_delete: If you delete your backup codes you will no longer be able to
         use them to sign in.
       confirm_delete: Are you sure you want to delete your backup codes?
       generate: Get codes
       last_code: You used your last backup code. Please print, copy or download the
         codes below. You can use these new codes the next time you sign in.
       regenerate: Get new codes
-      subinfo_html: '<b>Don’t lose these codes</b>. Download, print, or copy them.
-        Each code can only be used once. After you’ve used all 10 codes, we’ll
-        give you 10 new codes.  Keep your codes as safe as your password.'
-      subtitle: Your backup codes
+      saved: I’ve put my backup codes in a safe place.
+      subinfo_html: '<b>If you lose your device, you’ll need these codes to sign into
+        %{app_name}.</b> Save or print them and put them somewhere safe.'
+      subtitle: Save these backup codes
       title: Backup codes
```

Comment:
I think it's more like a heading, but this is an old key (the key name has been kept the same) with a minor alteration to the text itself. For that reason, I left it as subtitle.

Changing the text itself for title, and merged two keys, seems more like a content/UX change.

Reply:
Let's bring this up at next retro, I generally think the page heading should always match the browser tab title as a default, and in cases where it's not, then we should expect that as part of the initial design hand-off, so that we're not getting into a habit of fixing these things after the fact.

At the very least, I think changing `subtitle` to `heading` would be an improvement.

Reply:
Changed key name.